Games/types/Mafia/roles/cards/BreadGiver.js

```diff
@@ -8,47 +8,20 @@ module.exports = class BreadGiver extends Card {
     this.meetings = {
       "Give Bread": {
         states: ["Night"],
-        flags: ["voting"],
+        flags: ["voting", "multi"],
+        multiMin: 2,
+        multiMax: 2,
```

Comment:
this fix prevents baker from giving the same person bread twice, it was intentionally this way unless you want it otherwise